docs/crio.8.md

```diff
@@ -166,6 +167,8 @@ For the bind-mount conditions, only mounts explicitly requested by Kubernetes co
 
 **--log-level, -l**="": log crio messages above specified level: debug, info, warn, error (default), fatal or panic
 
+**--log-filter**="": filter the log messages by the provided regular expression
+
```

Comment:
Can you add an example for the filter in play?  It would be nice to have at least a simple RE that would work and would get the user a step up.  Ditto the next man page.

Reply:
Yes, added an example about request logging.